StardewCropCalc/src/com/elizabethburgessamt/stardewcropcalc/fields/Field.java

```diff
@@ -0,0 +1,8 @@
+package com.elizabethburgessamt.stardewcropcalc.fields;
+
+public interface Field {
+	

```

Comment:
This should be an abstract class, not an interface. Personally I like the rule that an abstract class indicates that a child class is still something.  For instance, a cat is an animal, but there are no concrete versions of the animal class, while there are instances of the cat class.